jemquarie.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_runtime_dependency "activesupport", '~> 6.1', '>= 6.1.4.7'
+  spec.add_runtime_dependency "rack", '~> 2.2.0', '< 3.0.0'
```

Comment:
You are adding a runtime dependency to make sure that our actual dependency `httpi` is working as expected, because `httpi` in version `2.5.0` doesn't specify the `rack` dependency correctly (they do `rack >= 0` for that [version](https://rubygems.org/gems/httpi/versions/2.5.0)). This is fine to me, but should only be temporary, as our gem here doesn't actually have a dependency to `rack` directly.

Interestingly, `httpi` in v3 does not have a dependency on `rack` anymore, and I wonder if we could try to upgrade to that version instead.

Anyhow, for now this is fine, **but** the version spec above, using the [twiddle-waka](https://thoughtbot.com/blog/rubys-pessimistic-operator), is overspecified. This should be enough:

```ruby
  spec.add_runtime_dependency "rack", '~> 2.2' # same as '>= 2.2, < 3'
```

Reply:
Ahh thanks for that. Now I know what that `~` means.